sdk/communication/azure-communication-rooms/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Release History
 
+## 1.1.0 (2024-04-15)
+
+- General Availability version of PSTN dial-out capability feature
+
 ## 1.1.0-beta.2 (Unreleased)
```

Comment:
You can remove the ## 1.1.0-beta.2 (Unreleased) and header below as this is not a release version. Its added automatically.

Reply:
unfortunately the CI fails if the header is not included for the beta.2 version. I removed the header but had to add it back in :) 
<img width="840" alt="image" src="https://github.com/Azure/azure-sdk-for-java/assets/113146208/adb799d8-090b-4cd7-a3a4-6a329637ef06">

Reply:
This will affect the changelog and later on release note. I believe this is checked against another version configured elsewhere. One way is to check the commit this beta2 is added to see any other files are being updated in the same commit.

Reply:
I think you need to modify this client_version.text to use 1.1.0 instead of 1.1.0-beta2. 
Then you should be able to update the POM.xml, previously reverted, to reference 1.1.0

https://github.com/Azure/azure-sdk-for-java/blob/main/eng/versioning/version_client.txt

You can refer to this commit made by Muhammad when releasing Rooms GA SDK.

https://github.com/Azure/azure-sdk-for-java/commit/46c4a5b6c5d5e1b96cc9b34d8a5d303113754a40